app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/models/db/DataOpenHelper.kt

```diff
@@ -0,0 +1,689 @@
+        if (version == 44) {
+            val columnNames = getColumnNames(db, "episode")
+            if (!columnNames.contains("show_notes")) db.execSQL("UPDATE episode SET show_notes = NULL")
+        }
+    }
+
+    private fun addDefaultPlaylists(db: SQLiteDatabase) {
+        val insert = "INSERT INTO playlists (uuid, title, unplayed, partiallyPlayed, finished, audioVideo, allPodcasts, downloaded, downloading, notDownloaded, autoDownload, autoDownloadWifiOnly, autoDownloadIncludeHotspots, autoDownloadPowerOnly, sortId, iconId, starred, syncStatus, sortPosition, filterHours) VALUES "
+        db.execSQL("$insert('2797DCF8-1C93-4999-B52A-D1849736FA2C', 'New Releases', 1, 1, 0, 0, 1, 1, 1, 1, 0,  0,  0,  0,  0, 10,  0,  1,  1,  336);")
+        db.execSQL("$insert('D89A925C-5CE1-41A4-A879-2751838CE5CE', 'In Progress',  0, 1, 0, 0, 1, 1, 1, 1, 0,  0,  0,  0,  0, 23,  0,  1,  2,  0);")
+        db.execSQL("$insert('78EC673E-4C3A-4985-9D83-7A79C825A359', 'Starred',      1, 1, 1, 0, 1, 1, 1, 1, 0,  0,  0,  0,  0, 39,  1,  1,  3,  0);")
+
+//        // 1. unplayed
+//        // 2. partiallyPlayed
+//        // 3. finished
+//        // 4. audioVideo
+//        // 5. allPodcasts
+//        // 6. downloaded
+//        // 7. downloading
+//        // 8. notDownloaded
+//        // 9. autoDownload
+//        // 10. autoDownloadWifiOnly
+//        // 11. autoDownloadIncludeHotspots
+//        // 12. autoDownloadPowerOnly
+//        // 13. sortId
+//        // 14. iconId
+//        // 15. starred
+//        // 16. syncStatus
+//        // 17. sortPosition
+//        // 18. filterHours
+
+//        Old playlists
+//        String insert = "INSERT INTO playlists (uuid, title, unplayed, partiallyPlayed, finished, audioVideo, allPodcasts, downloaded, downloading, notDownloaded, autoDownload, autoDownloadWifiOnly, autoDownloadIncludeHotspots, autoDownloadPowerOnly, sortId, iconId, starred, syncStatus, sortPosition) VALUES ";
+//        db.execSQL(insert + "('ad036f20-1118-0130-36a2-60f8472cfba6', 'Unplayed',   1, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0,  8, 0, 1, 1);");
+//        db.execSQL(insert + "('ad036980-1118-0130-36a2-60f8472cfba6', 'Audio',      1, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 27, 0, 1, 2);");
+//        db.execSQL(insert + "('ad036d60-1118-0130-36a2-60f8472cfba6', 'Video',      1, 1, 0, 2, 1, 1, 1, 1, 0, 0, 0, 0, 0, 31, 0, 1, 3);");
+//        db.execSQL(insert + "('ad0370b0-1118-0130-36a2-60f8472cfba6', 'Downloaded', 1, 1, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 15, 0, 1, 4);");
+    }
+
+    private fun getColumnNames(db: SQLiteDatabase, tableName: String): List<String> {
+        val names: MutableList<String> = ArrayList()
+        var cursor: Cursor? = null
+        try {
+            cursor = db.rawQuery("select * from $tableName limit 1", null)
+            if (cursor != null) names.addAll(listOf(*cursor.columnNames))
+        } catch (e: Exception) {
+            Timber.e(e)
+        } finally {
+            cursor?.close()
+        }
```

Comment:
Kotlin has `Closable.use()` [extension](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html) that can be used here to avoid `try/finally`. You'll still need to catch the exception and log it though. This applies to all `Cursor` uses in this file.